modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -343,6 +343,12 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         static Ptr<ReLULayer> create(const LayerParams &params);
     };
 
+    class CV_EXPORTS ReLU6Layer : public ActivationLayer
```

Comment:
can we make the layer slightly more universal? `result(x, y, c) = min(max(src(x, y, c), a), b)` with customizable `a` and `b`?

Reply:
Generalized. Name of layer is the same.